packs/victorops/actions/open_incident.py

```diff
@@ -0,0 +1,17 @@
+import urllib
+from lib.action import VictorOpsAction
+import json
+import time
+
+class OpenIncident(VictorOpsAction):
+  def run(self, severity, entity, message=None):
+    prms = {
+    "message_type": severity,
+    "timestamp": int(time.time()),
+    "entity_id":entity,
+    "state_message":message
+    }
+    post_data = json.dumps(prms).encode()
+    data = urllib.urlopen(self.url, post_data)
```

Comment:
Please use `request` package (`request.post`). There are multiple benefits over using a built-in urllib:

1. No issues with differences between Python 2.x and 3.x
2. request perform ssl cert validation